cmd/internal/cli/push.go

```diff
@@ -60,20 +82,34 @@ var PushCmd = &cobra.Command{
 	Args:                  cobra.ExactArgs(2),
 	PreRun:                sylabsToken,
 	Run: func(cmd *cobra.Command, args []string) {
-		handlePushFlags(cmd)
+		file, dest := args[0], args[1]
```

Comment:
Could we put the code for the switch statements in separate functions so we can write unit tests for error cases later on? I can see that the library and ORAS cases start to be a little lengthy.

Reply:
Same comment as with the pull switch statement. Let's create some tech-debt issues before we merge this PR. Then follow up and fix